### PR TITLE
Styleupdate ondat

### PR DESCRIPTION
**Pull request by tlssuse**
Hi, @dariavladykina: This existing document has been converted to use the new TRD stylesheet with a few other minor content updates.  Please review and, when ready, flag for publication.  Thank you. ~ Terry

### CODE REVIEW

File: kubernetes/start/ondat/adoc/gs_rancher_ondat.adoc [hunk 1]
```diff
@@ -387,18 +463,18 @@ storageos-cli-77748d9c84-dtbg7
...
-. simulating a node failure scenario to demonstrate how Ondat's replication feature can to quickly fail over to a new replica volume when a master volume suddenly goes permanently offline.
+. simulating a node failure scenario to demonstrate how {otherproduct1}'s replication feature can to quickly fail over to a new replica volume when a master volume suddenly goes permanently offline.
 
 
 === Deploy a stateful application
 
 For this demonstration, you will deploy MongoDB as the stateful application, using the https://github.com/percona/percona-server-mongodb-operator[MongoDB operator] developed and maintained by Percona.
 
-. Deploy the  operator with `kubctl`.
+. Deploy the  operator with `kubectl`.
```
Comment (dariavladykina) [inline kubernetes/start/ondat/adoc/gs_rancher_ondat.adoc:477]:
```suggestion
. Deploy the operator with `kubectl`.
```

File: kubernetes/start/ondat/adoc/gs_rancher_ondat.adoc [hunk 2]
```diff
@@ -623,11 +705,11 @@ Also notice that a new replica volume was automatically created on __demo-worker
 
 In this guide, you:
 
-* deployed Ondat into a Kubernetes cluster in your SUSE Rancher cloud native landscape.
+* deployed {otherproduct1} into a Kubernetes cluster in your {productname} landscape.
 * created a custom StorageClass, which enabled replication and Topology-Aware Placement (TAP) to guarantee two replica volumes distributed on nodes in different availability zones.
 * demonstrated how Ondat can automatically detect an offline master volume, elect a replica volume to become master, and create a new replica on a different node to maintain the defined configuration.
 
-With Ondat and SUSE Rancher, you can create a Kubernetes native stack that ensures your critical, stateful workloads are highly available and durable.
+With Ondat and {platform1}, you can create a Kubernetes-native stack that ensures your critical, stateful workloads are highly available and durable.
```
Comment (dariavladykina) [inline kubernetes/start/ondat/adoc/gs_rancher_ondat.adoc:712]:
```suggestion
With {otherproduct1} and {platform1}, you can create a Kubernetes-native stack that ensures your critical, stateful workloads are highly available and durable.
```

**Review by dariavladykina — APPROVED** [review]

**Review by tlssuse — COMMENTED** [review]
Perfect!  Thank you for catching these, @dariavladykina .

**Comment by tlssuse** [comment]
> Perfect! Thank you for catching these, @dariavladykina .

I made one more change, setting the ROLE in the DC file.

Please merge into main when you are ready.

**Pull request merged**